src/custom_interactions/chat_input/manage_user_identity.ts

```diff
@@ -382,6 +389,9 @@ export default new CustomInteraction({
                         '```json',
                         JSON.stringify(db_user_data_update_document, null, 4),
                         '```',
+                        '',
+                        '**REASON:**',
+                        `\`\`${reason}\`\``,
```

Comment:
```suggestion
                        '```',
                        reason,
                        '```',
```